src/cli.rs

```diff
@@ -337,6 +337,14 @@ pub fn add_global_flags<'a, 'b>(app: App<'a, 'b>) -> App<'a, 'b> {
             .takes_value(true)
             .default_value("ssl://electrum.blockstream.info:60002"),
     )
+    .arg(
```

Comment:
Here in cli I suggest doing with runtime feature checking (since the `arg` stuff doesn't require feature-specific struct) like I did in 3c8b8e4fca89c5341824ddda6ee55a5d9cf90717
I think compile checking would bloat the code too much, in this case runtime check is fine in my opinion

Reply:
I included electrum `server` parameter under a runtime feature check, now it's not useful since electrum is the default, but it could probably not be the default one day

Reply:
Yeah, I also like the checks with `cfg!()` better in this case